src/or/dirvote_common.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright (c) 2018, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+/**
+ * \file dirvote_common.h
+ * \brief Header file for dirvote_common.c.
```

Comment:
See comment previously on shared_random_common.c about why I don't like the "common" name here.

This module could probably be split up into the part about consensus schedules; the part about signatures; and the part about authority certs.  But it would make sense to do that _after_ this branch is merged, so let's open a ticket to remember?

Reply:
Yes